bigquery/params.go

```diff
@@ -306,14 +363,13 @@ func paramValue(v reflect.Value) (*bq.QueryParameterValue, error) {
 		return res, nil
 
 	case typeOfRat:
-		// big.Rat types don't communicate scale or precision, so we cannot
```

Comment:
HOORAY

Reply:
Humm, I need to bring back this comment, because still if you pass a `*big.Rat`, it is going to be converted to a NUMERIC type. For BIGNUMERIC you have to use the new `QueryParameterValue`